presto-main/src/main/java/com/facebook/presto/execution/FailedQueryExecution.java

```diff
@@ -222,7 +246,7 @@ public void pruneInfo()
     @Override
     public Optional<ResourceGroupId> getResourceGroup()
     {
-        return resourceGroup;
+        return Optional.empty();
```

Comment:
Don't all failed queries become a FailedQueryExecution (including ones with resource groups)?

Reply:
It makes the code a bit messier. How important is it to have this?